terraform/modules/hub/static_ingress.tf

```diff
@@ -251,3 +251,280 @@ resource "aws_lb_listener" "static_ingress_https" {
     target_group_arn = aws_lb_target_group.static_ingress_https.arn
   }
 }
+
+resource "aws_lb" "static_ingress_fargate" {
+  name                             = "${var.deployment}-static-ingress-fargate"
+  load_balancer_type               = "network"
+  internal                         = false
+  enable_cross_zone_load_balancing = true
+  subnets                          = [aws_subnet.public.*.id]
```

Comment:
`aws_subnet.public` resource doesn't seem to exist

Reply:
Good spot! Should be `aws_subnet.ingress.*.id`. Updated and squashed.